lib/chef/knife/tidy_server_report.rb

```diff
@@ -39,6 +39,11 @@ def run
           version_count = cookbook_count(cb_list).sort_by(&:last).reverse.to_h
           used_cookbooks = {}
           nodes = nodes_list(org)
+          db_nodes = rest.get("/organizations/#{org}/nodes")
+          unless nodes.length == db_nodes.length
+            Chef::Log.error("Search index is out of data for organization #{org}. Skipping report.")
```

Comment:
s/data/date/ ?

Reply:
I think also our convention thus far in knife-tidy has been to use `ui.x` so perhaps we need a `ui.error` message here instead of `Chef::Log.error`

Reply:
Those small nit picks aside. This looks good!!